.azure-pipelines/run-test-elastictest-template.yml

```diff
@@ -119,6 +119,10 @@ parameters:
     type: string
     default: ""
 
+  - name: PARALLEL_MODES_FILE
+    type: string
+    default: "default.json"
```

Comment:
It is recommended to leave this parameter empty, as parallel run is disabled by default and we already have a default value in test_plan.py and  elastictest. Additionally, consider adding a comment for clarification if necessary.

Reply:
Good idea! Let me fix this